lib/ViewModels/SettingsPanelViewModel.js

```diff
@@ -22,6 +22,15 @@ var SettingsPanelViewModel = function(options) {
     this.baseMaps = defaultValue(options.baseMaps, []).slice();
     this.mouseOverBaseMap = undefined;
 
+    var that = this;
+    if (defined(options.baseMapName)) {
+        this.baseMaps.map (function(item) {
+            if (options.baseMapName === item.catalogItem.name) {
+                that.terria.baseMap = item.catalogItem;
```

Comment:
> I started with it in index.js, but then it's only available it you build that in a certain way.

Yeah, but you're still requiring that index.js do something specific (i.e. use the settings panel).  If you're going to impose a requirement on applications, it's much better for it to be a non-UI requirement.